portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -267,14 +267,44 @@ definition {
 	}
 
 	@description = "Verify that a help text can be added to a Color Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133142 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		FormViewBuilder.setFieldHelpText(fieldLabel = "Color Help Text");
```

Comment:
@leilanyaragao, you created a new macro on `FormFields.macro`, but you're still invoking the one from `FormViewBuilder`, can you please fix it?